src/SearchResults.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import moment from "moment";
+
+function SearchResults(props) {
+  return (
+    <table className="table">
+      <TableHead />
+      <tbody>
+        {props.results.map((item, index) => {
+          const nightsSpent = moment(item.checkOutDate).diff(
+            item.checkInDate,
+            "days"
+          );
+          return (
+            <tr key={index}>
+              <td scope="row">{item.id}</td>
+              <td>{item.title}</td>
+              <td>{item.firstName}</td>
+              <td>{item.surname}</td>
+              <td>{item.email}</td>
+              <td>{item.roomId}</td>
+              <td>{item.checkInDate}</td>
+              <td>{item.checkOutDate}</td>
+              <td>{nightsSpent}</td>
+            </tr>
+          );
+        })}
+      </tbody>
+    </table>
+  );
+}
+
+function TableHead() {
```

Comment:
Good job here, this is exactly what I meant. Although it would be cleaner to have each component in it's own file, so I would move `TableHead` component to `TableHead.js` file and import it here. Then you could create `TableRow` component and extract lines 15 to 25 there, passing the data it needs to render as props.

Reply:
done.